apps/widget/cypress/e2e/notifications-list.spec.ts

```diff
@@ -191,18 +190,9 @@ describe('Notifications List', function () {
     scrollToBottom();
     cy.getByTestId('notification-list-item').should('have.length', 26);
 
-    cy.getByTestId('notification-list-item')
-      .should('exist')
-      .each(($item, $index) => {
-        const notificationContentNumber = 20 - $index > 0 ? ' ' + (20 - $index).toString() : '';
-        cy.wrap($item)
-          .invoke('text')
-          .then((text) => {
-            if ($index !== 0) {
-              expect(text).to.contains(`John${notificationContentNumber}`);
-            }
-          });
-      });
+    for (let i = 0; i < 21; i++) {
+      cy.getByTestId('notification-list-item').contains(`John ${i}`).should('exist');
+    }
```

Comment:
fixed Cypress flaky test... basically here we expected to have all notifications shown in the order, but without awaiting during creation they won't be ordered... but I didn't wanted to introduce the "pause" between the triggers to create this order, as it will make the tests running unnecessary longer